Source/WebCore/editing/WritingToolsCompositionCommand.cpp

```diff
@@ -40,9 +40,6 @@ WritingToolsCompositionCommand::WritingToolsCompositionCommand(Ref<Document>&& d
 
 void WritingToolsCompositionCommand::replaceContentsOfRangeWithFragment(RefPtr<DocumentFragment>&& fragment, const SimpleRange& range, MatchStyle matchStyle, State state)
 {
-    if (endingSelection().isNoneOrOrphaned() || !endingSelection().isContentEditable())
-        return;
```

Comment:
This seems like a good check to have, is there no way to keep it?

Reply:
it was incorrect, since the endingSelection isn't used at this point anyways, and becomes out of date